src/Tasks/Common/Resources/Strings.resx

```diff
@@ -381,8 +381,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1065: Cannot find app host for {0}. {0} could be an invalid runtime identifier (RID). For more information about RID, see https://aka.ms/rid-catalog.</value>
     <comment>{StrBegin="NETSDK1065: "}</comment>
   </data>
-  <data name="CannotUseAppHostWithoutRuntimeIdentifier" xml:space="preserve">
-    <value>NETSDK1066: A RuntimeIdentifier must be specified to publish a framework-dependent application with an application host.</value>
+  <data name="Unused1066" xml:space="preserve">
+    <value>NETSDK1066: </value>
```

Comment:
My intention for when we deleted codes was to delete them from resx/xliff and put them in:

https://github.com/dotnet/sdk/blob/99909b3075402d3e01045c9d76eecea056a8757c/src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenThatWeHaveErrorCodes.cs#L19-L23

Reply:
Cool, I missed that since I didn't look at the test's, only saw the test failure message and took action.  I may change the failure message to mention intentionally deleted errors should be added to the exclusion list here.

I'll amend with removing the message and adding it to the exclusion list, at least.